README.md

```diff
@@ -270,4 +270,65 @@ docker run -i --rm \
 86. `download_attachment` - Download an uploaded file from a GitLab project by secret and filename
 <!-- TOOLS-END -->
 
+#### Examples: Group-scoped Search
```

Comment:
it is weird to add this to readme while other tools has no such documentation. let's remove it